units/src/amount/mod.rs

```diff
@@ -128,6 +128,68 @@ impl Denomination {
     }
 }
 
+impl PartialOrd for Denomination {
+    fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
+        Some(self.cmp(other))
+    }
+}
+
+impl Ord for Denomination {
+    // This implementation is probably excessively precautions.
```

Comment:
```suggestion
    // This implementation is probably excessively precautious.
```

Reply:
I think just `cautious` is better. "precautious" might be a word but I've never heard it before :).

Reply:
Yes indeed, that is surely what my brain was trying to tell me.  I definitely didn't mean precocious.

Reply:
Sweet, this is mad, with two grammar nerds on this team I may finally up my grammar game to a level I'm happy with.

> is probably excessively precautions.

How embarrassment.  (Joke based on an Australian movie almost certainly lost on non-Australians.)

>  "precautious" might be a word but I've never heard it before :).

I'm chalking this up as a win, I had assumed the learning new words (excluding slang) was one directional from Andrew to me.

https://duckduckgo.com/?q=precautous+def&t=canonical&ia=web

I do note however that I still misspelled it in the search.